electron/ironfish/IronFishManager.ts

```diff
@@ -330,10 +330,7 @@ export class IronFishManager implements IIronfishManager {
 
   async saveNodeConfig(values: Partial<ConfigOptions>): Promise<void> {
     this.nodeSettings.setValues(values)
```

Comment:
I don't think this solves the problem. We want the node to be restarted after the settings change, but perhaps we just want to force reload of the window instead of these manual commands here?

Reply:
Yes, you are right after looking though code I thought that restart is no required but its required.
Add wait for shutdown of the node after `initStatus` changes restart is triggered in Initializing component
https://github.com/iron-fish/node-app/blob/bug/IFL-898/src/routes/Initializing.tsx#L78